### PR TITLE
chore: Increase code coverage on JSON FieldReader and FieldWriter

### PR DESCRIPTION
**Pull request by joeylimd**
Relates to: #202

### CODE REVIEW

File: runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldReaderTest.java [hunk 1]
```diff
@@ -748,4 +752,821 @@ private void resetField(JsonField field) {
         field.setValue(null);
         field.setFieldType(null);
     }
+
+    @Test
+    public void testJsonFieldDoubleMax() throws Exception {
+        String filePath = "src" + File.separator + "test" + File.separator + "resources"
```
Comment (rohanmars) [inline runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldReaderTest.java]: Could we make some private/protected methods for repetitious code? There seems to a few standard code patterns in use in the file.

Comment (joeylimd) [reply runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldReaderTest.java]: Done. extracted private method for repetitive codes. Please review. Thanks.

File: runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldWriterTest.java [hunk 2]
```diff
@@ -626,4 +635,565 @@ private String prettyPrintJson(String json) throws IOException {
...
+        AtlasInternalSession session = read(path, outputFieldType, fieldPath);
+
+        assertEquals(expectedValue, session.head().getSourceField().getValue());
+        assertEquals(0, session.getAudits().getAudit().size());
+    }
+
+    @Test
+    public void testJsonFieldFloatMaxRangeOut() throws Exception {
+        Path path = Paths.get("target" + File.separator + "test-write-field-float-max-range-out.json");
+        String fieldPath = "/primitiveValue";
```
Comment (rohanmars) [inline runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldWriterTest.java]: Changes look good Joey. Can you put these values directly on the method call, there is not need to declare a variable for these. Similar to what you have done the JSONFieldReader

Comment (joeylimd) [reply runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldWriterTest.java]: Done. Please review. Thanks

**Comment by rohanmars** [comment]
LGTM :+1:

**Pull request merged**